client/src/components/UI/StarRating/StarRating.js

```diff
@@ -1,21 +1,51 @@
-import React from "react";
+import React, { useState } from "react";
+import axios from "axios";
 
 const starRating = props => {
-    let stars = [];
-    for (let i = 0; i < 5; i++) {
-      let klass = "star-rating__star";
```

Comment:
maybe className or somehow else? To name via transliteration not ok

Reply:
done